### DISCUSSION TITLE
Integration of OpenCV and JuliaImages (Discussion thread, GSoC)

### DISCUSSION THREAD

**Discussion by rushabh-v** [JuliaImages/Images.jl#946] [JSoC]
Hello everyone,

I am Rushabh Vasani, a 3rd-year undergrad from India. I am interested in the "Integration of OpenCV and JuliaImages" project idea and I have experience in developing Linux-based systems.

After following [this guide](https://docs.opencv.org/master/d8/da4/tutorial_julia.html), I was able to successfully install the OpenCV bindings for Julia on my local system. But I'm facing issues while using some of the functionalities (calling the functions, results in errors). Trying to figure it out.

I have gone through BinaryBuilder's documentation [here](BinaryBuilder) and trying to understand it better from other resources. Is there anything else I can do, to understand the project better? I am thinking of installing the bindings on a docker container once.

Thanks, Looking forward!

cc: @timholy @zygmuntszpak

**Comment by timholy** [comment]
Welcome Rushabh, glad to hear of your interest!

With regards to Julia/OpenCV integration, I would ping @archit120, who is the author of that blog post and who knows much more about this than me. He may yet have plans to tackle that integration, but I am not certain. Archit, any thoughts?

**Comment by archit120** [comment]
Hi Rushabh and Tim. 

I'd be happy to help you out with fixing the issues on your local OpenCV for Julia installation. Were you able to get the included tests to work? They can be run using `make opencv_test_julia`. However, I must add that I am planning to continue working on the integration of OpenCV with Julia and will _most probably_ be submitting a proposal for the same in the coming GSoC season.

**Comment by rushabh-v** [comment]
Hi Archit and Tim, Thanks for your responses.

Yes, I will try running the tests and let you know if I face any issues.

And thanks for letting me know about your plans of continuing this project as I believe you are much more qualified for this project than me. I'll try picking up other ideas :)

Good Luck!
